### ISSUE TITLE
Video URLs in API docs do not exist

### ISSUE THREAD

**Issue by IvanSanchez** [mapbox/mapbox-gl-js#3267]
The documentation for `VideoSource` at https://www.mapbox.com/mapbox-gl-js/api/#VideoSource refers to two video URLs, specifically:

```
   url: [
       'https://www.mapbox.com/videos/baltimore-smoke.mp4',
       'https://www.mapbox.com/videos/baltimore-smoke.webm'
   ],
```

Those two videos do not exist (my browser says 404), so I suggest changing the docs to point to working video URLs.

**Comment by 1ec5** [comment]
These videos moved to https://www.mapbox.com/blog/assets/baltimore-smoke.mp4 and https://www.mapbox.com/blog/assets/baltimore-smoke.webm.

**Comment by mollymerp** [comment]
Fixed in #3721
